openapi-cli/src/main/java/io/ballerina/openapi/cmd/OpenApiCmd.java

```diff
@@ -58,6 +59,9 @@
     @CommandLine.Option(names = {"-i", "--input"}, description = "Generating the client and service both files")
     private boolean inputPath;
 
+    @CommandLine.Option(names = {"--prefix"}, description = "Location of the file which contains any prefix to add")
```

Comment:
What about rewording like `Location of file containing <abc> prefix to add`
What do you mean by any prefix? What are we prefixing here? Looking at the description I am not clear what kind of prefix this should be.

Reply:
Updated the name to `license`. Please review.